src/libraries/System.IO.FileSystem/tests/File/Copy.cs

```diff
@@ -365,7 +365,7 @@ public void WindowsAlternateDataStreamOverwrite(string defaultStream, string alt
     /// <summary>
     /// Single tests that shouldn't be duplicated by inheritance.
     /// </summary>
-    [ActiveIssue("https://github.com/dotnet/runtime/issues/40867", TestPlatforms.Browser)]
+    [PlatformSpecific(~TestPlatforms.Browser)]
```

Comment:
I think it would be nice to add a small comment about why this is disabled on browser since it's not obvious. Maybe also keep the link to the issue since it has more context.